.github/workflows/packaging.yml

```diff
@@ -801,7 +803,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python_version: ["3.9", "3.10", "3.11", "3.12"]
+        python_version: ["3.9", "3.10", "3.11", "3.12", "3.13"]
```

Comment:
Unrelated to this PR but SPEC 0 has dropped 3.9 for some time, and 3.10 recently. Maybe we don't need to build all of these?

https://scientific-python.org/specs/spec-0000/

Reply:
PyArrow still builds for 3.9 so I think we can keep it for now. It's also still in security support for a while.